website/docs/contributing/YYYY-MM-DD.blog-template.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: url-slug-of-post # the URL slug of the blog post with dashes instead of spaces
+title: Blog post template
+author: Firstname Lastname
+author_url: # link to website or GitHub account
+author_image_url: # link to author profile image (optional – please remove this line if you don't want to use an author image. See below for more info)
+description: >
+  This is a short description of a FerretDB blog post.
+keywords:
+  - keyword1
+  - keyword2
+image: /img/blog/postgresql.png
+tags: [tag1, tag2]
```

Comment:
We can't add random tags, it has to be pre-existing one. I think a bit more guide here on allowed tags would be great.

Reply:
This is already covered in the content writing process: https://github.com/FerretDB/FerretDB/blob/6956f3bf25f8b56dc1fb7c40660dfbacde03e612/website/docs/contributing/content-process.md#tags

Reply:
I think CI is failing because using non existing flag